docs/source/swift-scripting.md

```diff
@@ -247,12 +243,12 @@ Now, you're able to generate code from a debuggable Swift Package Manager execut
     swift run
     ```
     
-    >**NOTE**: If your package ever seems to have problems with caching, run `swift package clean` before `swift run` for a totally clean build. It is not recommended to do this by default since it substantially increases build time.
+    >**Note**: If your package ever seems to have problems with caching, run `swift package clean` before `swift run` for a totally clean build. It is not recommended to do this by default, because it substantially increases build time.
     
-4. Build your target. Since `swift run` is being called from within your target, all of the pieces of the environment, including `$SRCROOT`, will automatically be passed to the environment of the executable, and you don't have to worry about passing anything manually. 
+4. Build your target. Because `swift run` is called from within your target, all of the pieces of the environment, including `$SRCROOT`, are automatically passed to the environment of the executable, and you don't have to worry about passing anything manually. 
 
-Now, every time you build your project, this script will get called. Since Swift knows not to recompile everything unless something's changed, it should not have a significant impact on your build time. 
+Now, every time you build your project, this script gets called. Because Swift knows not to recompile everything unless something's changed, it should not have a significant impact on your build time. 
 
 ## Swift-specific troubleshooting
 
-- If at any point you start seeing a bunch of errors around `SecTaskLoadEntitlements` resulting in an immediate exit of the script rather than showing the permission prompt, validate that all the folders you're looking for exist and are at the *exact* path you think they are. You may have a typo in one of your paths. 
\ No newline at end of file
+If you encounter errors around `SecTaskLoadEntitlements` that result in an immediate exit of the script instead of showing the permission prompt, validate that all the folders you're looking for exist at the correct path.
```

Comment:
i'd like to throw something about typos in here since a typo is what me fight with this for an hour 🙃